rotkehlchen/api/rest.py

```diff
@@ -4984,6 +4984,25 @@ def add_tokens_to_spam(self, tokens: list[EvmToken]) -> Response:
                 )
                 AssetResolver.clean_memory_cache(token.identifier)
 
+                for balances in self.rotkehlchen.chains_aggregator.balances.get(
+                    chain=(blockchain := token.chain_id.to_blockchain()),
+                ).values():
```

Comment:
explain here what you do. Or in the function's docstring.

I do not understand why you are doing this here without a short explanation.